.prettierrc

```diff
@@ -0,0 +1,17 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 4,
+  "printWidth": 120,
+  "semi": true,
+  "singleQuote": true,
+  "jsxSingleQuote": false,
+  "importOrder": ["^[./]"],
+  "importOrderSeparation": true,
+  "importOrderSortSpecifiers": true,
```

Comment:
aren't these specific to the trivago plugin ? 
https://github.com/trivago/prettier-plugin-sort-imports